### PR TITLE
SECURITY: migrate policy to web site, update content 

### PR DESCRIPTION
**Pull request by srenatus**
It's easier to link to, and easier to find that way. Also, it's deliberately
not part of the versioned docs: we don't want anyone to find old information
when it's important.

Fixes https://github.com/open-policy-agent/opa/issues/4272.

Note that we won't win any style awards for this, but let's make CSS tweaking
a follow-up issue.

### CODE REVIEW

File: docs/website/static/security.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+              <li>A suggested embargo date for this vulnerability is chosen and a CVE
+                (Common Vulnerabilities and Exposures (CVE®)) is requested for the vulnerability,
+                through the GitHub Advisory database.</li>
+
+              <li>A prenotification may be published on the security announcements channels listed below,
+                providing information about affected projects, severity, and the embargo date.</li>
+
+              <li>On the embargo date, the announcement is published.
+                The changes are pushed to the public repository and new builds are deployed.
+                <!--Within 6 hours of the mailing list being notified, a copy of the advisory will be published on the Bytecode Alliance blog.-->
```
Comment (philipaconrad) [inline docs/website/static/security.html:120]:
```suggestion

```
Looks like a leftover fragment of the original security policy. Do we need to keep it?

**Comment by ashutosh-narkar** [comment]
Is it possible to get a preview of the page?

**Comment by srenatus** [comment]
https://deploy-preview-5094--openpolicyagent.netlify.app/security

**Comment by ashutosh-narkar** [comment]
The website content lgtm. We seem to have some detailed info about how the issue will be handled, who handles it etc. in the markdown file. The website content breaks down the overall process pretty well, I wonder if it would be worth preserving some of the existing content in the markdown file and then linking to the website from there.

**Pull request merged**